pages/apim/3.x/user-guide/publisher/service-discovery/service-discovery-eureka.adoc

```diff
@@ -4,4 +4,71 @@
 :page-layout: apim3x
 :page-title: Eureka service discovery
 
-include::https://raw.githubusercontent.com/gravitee-io/gravitee-service-discovery-eureka/master/README.adoc[]
+== Configuration
+
+The configuration is loaded from the common GraviteeIO Gateway configuration file (gravitee.yml) All configurations described by official Eureka discovery client documentation are available on the official github repository: https://github.com/Netflix/eureka/wiki. Gravitee Gateway uses Eureka Discovery client v1 and fetch configured Eureka Servers. It does not register the gateway application into Eureka registry. Please refer the official Eureka documentation for advanced uses and advanced concepts.
+
+For more information, see the complete link:{{ 'https://github.com/gravitee-io-community/gravitee-service-discovery-eureka/blob/master/README.adoc' }}[Eureka Client Config].
+
+== Enable Eureka Service Discovery in Gravitee.io APIM
+
+Now that your configured your Eureka service, go to APIM console:
+
+* Create or select an existing API.
+* Under API's submenu, click *Proxy*.
+* Under *Backend Service*, select *Endpoints*.
+
+image::{% link images/apim/3.x/api-publisher-guide/service-discovery/service-discovery-endpoints.png %}[]
+
+* Click on the *Edit Group* icon.
+* Select *SERVICE DISCOVERY* tab.
+* Check *Enabled service discovery* checkbox.
+* For *Type* dropdown list, select *Eureka Service Discovery*.
+* For *Application* enter your application name.
+
+image::{% link images/apim/3.x/api-publisher-guide/service-discovery/service-discovery-eureka-configuration.png %}[]
+
+* Click *SAVE*.
+
+Your API should now appear out of sync in the top banner. Be sure to click *deploy your API*.
+
+image::{% link images/apim/3.x/api-publisher-guide/service-discovery/service-discovery-redeploy.png %}[]
+
+NOTE: Endpoints configured through the APIM console before service discovery was enabled are not removed. The Gravitee.io gateway will continue to consider those endpoints in addition to the ones discovered through Eureka integration. The endpoints dynamically discovered through Eureka are not displayed in APIM console. You can remove the endpoints define through the APIM console. However, we encourage you to keep at least one endpoint declared as secondary. Secondary endpoints are not included in the load-balancer pool and are only selected to handle requests if Eureka is no longer responding.
```

Comment:
```suggestion
NOTE: Gravitee Gateway will not remove endpoints configured through the APIM console before service discovery was enabled - it will continue to consider such endpoints in addition to the ones dynamically discovered through Eureka integration (these are not shown in the APIM console). You can manually remove any endpoints defined through the APIM console. However, it is highly recommended that you keep at least one such endpoint declared as secondary - secondary endpoints are not included in the load-balancer pool and are only selected to handle requests if Eureka is no longer responding.
```